docs/docs/types/message.md

```diff
@@ -0,0 +1,48 @@
+Focused on deliverying outbound communication. Perhaps announcing a product launch, new content, top-level alert, etc. Message modules are simple text modules which can be used in a variety of formats.
```

Comment:
Jumping into this page lacks context. I think "The Message type focuses on delivering outbound communication".